sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -491,6 +497,223 @@ public interface IBar<T>
             UcfgVerifier.VerifyInstructions(code, "Foobar");
         }
 
+        [TestMethod]
+        public void Invocations_MethodsWithDefaultParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo()
+        {
+            SendEmailAsync(""body"");                   // %0 := Namespace.Class1.SendEmailAsync(string, bool) [ this const ]
+            SendEmailAsync(""body"", isHtml: true);     // %1 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+            SendEmailAsync(""body"", isHtml: false);    // %2 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+        }
+
+        public System.Threading.Tasks.Task SendEmailAsync(string body, bool isHtml = false)
+        { return null; }
+    }
+}";
+            UcfgVerifier.VerifyInstructions(code, "Foo");
+        }
+
+        [TestMethod]
+        public void Invocations_UsedNamedParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo()

```

Comment:
[important] I would suggest to add an argument/parameter of type `string` to `Foo` (e.g. `Foo(string s)`) and use it instead of the constant string in your tests. This would allow to check whether (or not) we are passing them in the right order to the method call.